src/api/backend.js

```diff
@@ -12,14 +13,29 @@ const getApi = () => {
   return api;
 };
 
-export const getAddressLLM = async (walletAddress) => {
+export const getUsersByAddress = async (blockchainAddress) => {
   const api = getApi();
 
   const { data } = await api.get("/users", {
     params: {
-      blockchainAddress: walletAddress,
+      blockchainAddress,
     },
   });
 
-  return data;
+  return data.map(({ uid, merits }) => ({
+    uid,
+    merits: ethers.utils.parseUnits(merits.toFixed(12), 12),
+  }))
 };
+
+export const addMeritTransaction = async (userId, amount) => {
+  const api = getApi();
+
+  const formattedAmount = ethers.utils.formatUnits(amount, 12);
+  await api.post("/merit-transactions", {
+    userId,
+    amount: formattedAmount,
```

Comment:
this might need to be minus amount ?

Reply:
Minus is added here: https://github.com/liberland/liberland_frontend/pull/99/files#diff-372c7d46a63791bfa3dbddf18062c2e107720f213d4e260f16eef16f0ac21044R52